shard.yml

```diff
@@ -0,0 +1,10 @@
+name: coala-bears
+version: 0.12.0.dev99999999999999
```

Comment:
@jayvdb, I am a bit confused here about the name, version, authors, etc. This `version` part is mandatory according to the shard, BTW.

Reply:
I think you have the right content for those. But they need to come from a template